alert-service/src/main/resources/application.yml

```
@@ -0,0 +1,42 @@
+    properties:
+      hibernate:
+        format_sql: true
+        dialect: org.hibernate.dialect.PostgreSQLDialect
+
+  kafka:
+    url: ${KAFKA_URL}
+    bootstrap-servers: ${KAFKA_URL}
+    producer:
+      key-serializer: org.apache.kafka.common.serialization.StringSerializer
+      value-serializer: org.apache.kafka.common.serialization.StringSerializer
+      retries: 5 # 메시지 전송 실패시 재시도 횟수 5회.
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)